tests/helpers/test_selector.py

```diff
@@ -453,6 +453,11 @@ def test_select_selector_schema_error(schema):
             ("friendly_name", "device_class"),
             (None,),
         ),
+        (
+            {"entity_id": "sensor.abc", "exclude_attributes": ["friendly_name"]},
+            ("device_class", "state_class"),
+            (None,),
```

Comment:
Excluded attributes should be invalid.

```suggestion
            (None, "friendly_name"),
```

Reply:
How would that work in the backend tests? As this is a frontend feature?
Nothing is filtered here, it's just configuration stubs.

Reply:
OK, I get the misunderstanding. Attributes are not banned, just not shown.

Maybe I should rename `exclude_attributes` to `hide_attribute`? Would that make more clear?